rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/ConsensusRule.java

```diff
@@ -18,7 +18,8 @@
 
 package org.ethereum.config.blockchain.upgrades;
 
-public enum ConsensusRule {
```

Comment:
Was this change on purpose?

Reply:
I think it was an accidental new line, but should not be like that.

Reply:
Thanks guys, I will resolve it